src/components/ContestDetails.css

```diff
@@ -26,5 +26,9 @@
 .photo-drawer img {
   width: 100%;
   height: auto;
-  box-shadow: rgba(0, 0, 0, 0.4) 2px 10px 9px;
-}
\ No newline at end of file
+  box-shadow: rgba(0, 0, 0, 0.6) 2px 10px 9px;
+}
+
+.back-button {
+  border: 1px solid #DDD ;
```

Comment:
😬 Not a fan of this rule, can you revert it?
I don't see a reason to have 1 button be different from the others in terms of border.